kokain-ksp/src/test/java/com/schwarz/kokain/ksp/KokainProcessorKotlinTest.kt

```diff
@@ -0,0 +1,89 @@
+package com.schwarz.kokain.ksp
+
+import com.tschuchort.compiletesting.KotlinCompilation
+import com.tschuchort.compiletesting.SourceFile
+import com.tschuchort.compiletesting.symbolProcessorProviders
+import org.junit.Assert
+import org.junit.Test
+
+class KokainProcessorKotlinTest {
+
+    @Test
+    fun testKotlinBasicGeneration() {
+
+        val subEntity = SourceFile.kotlin(
+            "FooBean.kt",
+            HEADER +
+                "@EBean\n" +
+                "@EFactory\n" +
+                "open class FooBean {\n" +
+                "}"
+        )
+
+        val compilation = compileKotlin(subEntity)
+
+        Assert.assertEquals(compilation.exitCode, KotlinCompilation.ExitCode.OK)
+    }
+
+//    @Test
```

Comment:
why is this commented out?
can we fix it or use disable it with a comment why it is currently not being run?